lib/office_search.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module OfficeSearch
+  def self.search_by_location(near, opts)
```

Comment:
Now this method has been moved into the `OfficeSearch` class,  perhaps it can be shortened to `by_location`..?

Reply:
good shout